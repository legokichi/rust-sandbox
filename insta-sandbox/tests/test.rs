use rstest::*;

#[rstest]
#[case(0, 0)]
#[case(1, 1)]
#[case(2, 1)]
#[case(3, 2)]
#[case(4, 3)]
#[rstest]
fn test_hello_world(#[case] input: u32, #[case] expected: u32) {
    insta::assert_debug_snapshot!(vec![input, expected]);
}