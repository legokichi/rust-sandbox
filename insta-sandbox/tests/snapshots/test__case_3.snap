---
source: tests/test.rs
expression: "vec![input, expected]"

---
[
    2,
    1,
]
